src/apps/subjects/db/schemas.py

```diff
@@ -16,6 +16,7 @@ class SubjectSchema(Base):
     first_name = Column(StringEncryptedType(Unicode, get_key), nullable=False)
     last_name = Column(StringEncryptedType(Unicode, get_key), nullable=False)
     nickname = Column(StringEncryptedType(Unicode, get_key), default=None, nullable=True)
+    tag = Column(String, default=None, nullable=True)
```

Comment:
This might too early to consider optimizations, but it sounds like managers are often going to be filtering subjects by tag; does it make sense to add an index for it?

Reply:
Excellent point @farmerpaul. I think it's better to research and implement in a separate ticket. Not super familiar in this area, but there are several strategies we have available to choose from to performantly support text searches, filtering, etc. The one that keeps coming up after some quick googling is Postgres' [trigram indexing](https://sourcegraph.com/blog/postgres-text-search-balancing-query-time-and-relevancy) indexes, which would especially be helpful for supporting Search functionality.

Reply:
captured [M2-6251](https://mindlogger.atlassian.net/browse/M2-6251)